docs/poc-guidlines.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: default
+title: Firebolt POC Guidelines
+description: Make the necessary preparations for your Firebolt PoC
+nav_order: 12
+---
+
+# Firebolt POC Guidelines
```

Comment:
We want to avoid abbreviations in docs - do you want to say "proof of concept guidelines" or perhaps "trial guidelines?

Reply:
Sure - fixed